source/_integrations/wake_on_lan.markdown

```diff
@@ -60,11 +72,16 @@ This usually only works if the target device is connected to the same network. R
 The service to route the packet is most likely named "IP Helper". It may support Wake on LAN, but not all routers support this.
 {% endnote %}
 
-## Switch
+## Button
 
 The `wake_on_lan` (WOL) switch {% term integration %} allows you to turn on a [WOL](https://en.wikipedia.org/wiki/Wake-on-LAN) enabled computer.
 
-### Switch configuration
+The WOL button can only turn on your computer.
+It will send a magic packet to the mac-address specified in the configuration and as a button it is state-less meaning it can not monitor if the WOL enabled computer has actually received the wake-up call and has started.
```

Comment:
```suggestion
It will send a magic packet to the MAC address specified in the configuration. As a button, it is stateless. This means it can not monitor if the WOL-enabled computer has actually received the wake-up call and has started.
```